zendesk_apps_support.gemspec

```diff
@@ -19,7 +19,6 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'json'
   s.add_runtime_dependency 'image_size'
   s.add_runtime_dependency 'erubis'
-  s.add_runtime_dependency 'jshintrb', '~> 0.3.0'
```

Comment:
Nice, long past time we got rid of jshint

Reply:
:laugh